### ISSUE TITLE
Last follow is not verified   - 0.6.8 & 0.6.7 - Problems with following

### ISSUE THREAD

**Issue by ortegafernando** [InstaPy/InstaPy#5305]
I have found this: https://github.com/timgrossmann/InstaPy/issues/4983 and https://github.com/timgrossmann/InstaPy/issues/5034

And I have the same problem, but I cant reopen that issue.

Could anybody explain me how to solve it ?

```
INFO [2019-09-01 13:03:55] [xxxx] Ongoing Follow [2/30]: now following 'yy'...
WARNING [2019-09-01 13:04:35] [xxxx] Last follow is not verified. ~'xxxx' might be temporarily blocked from following

but I have verified that I have not the actions blocked.
```

May be adding "Message" in this list is enough ?
unfollow_util.py, line 1518:  post_action_text_correct = ["Following", "Requested"]

Or if I get "message" in the button does it really mean that I am blocked?

**Comment by Knrd2** [comment]
Im getting the exact same error.. any help please?

**Comment by jfranecki** [comment]
I believe Instapy is now 0.6.8, try updating and see if that helps.

**Comment by ortegafernando** [comment]
Hi, thanks a lot. I have update instapy. Same problem, same message.

May be I am really blocked. I will wait a few days to try it again and write here, thanks.

**Comment by meloluan** [comment]
Guys, I have the same problem, the bot can't follow anyone. I tested in version 0.6.7 and 0.6.8.

**Comment by savenelson** [comment]
"might be temporarily blocked from following." Same 
updated to 0.6.8

**Comment by rkoechel** [comment]
Try turning off the headless browser and see if you are really being blocked or not. I'm seeing this error too, but it seems like the issue is related to the browser window size and the script not finding what it's looking for in the DOM. When I've received this message, I have not been blocked.

**Comment by ortegafernando** [comment]
Hi @rkoechel  I have this option enabled since begining: 
```
headless_browser=False
```
After clicking in "follow button" I can see this: (text: message and an person icon)

![instapy](https://user-images.githubusercontent.com/8700791/74088812-b0367780-4a9a-11ea-9cd1-2361ae85810d.png)

So, what is the real problem? I dont know how exactly works instapy. Do you need me to get any debug log?

May be Instagram can show a "MESSAGE" text  ? So then whe have to change this line 616 in unfollow_util.py ? 

```
        elif following_status in ["Following", "Requested"]:
```
to
```
        elif following_status in ["Following", "Requested","Message"]:
```

?

Thanks

**Comment by rkoechel** [comment]
Ok, then you should be able to watch the browser as instapy runs. If the
follow button is available then you aren’t being blocked.

**Comment by ortegafernando** [comment]
Then, instapy can't detect that "message" button, can it ?

**Comment by ortegafernando** [comment]
Hi, anyone knows about this problem? Has anybody got this problem? Thanks

**Comment by ortegafernando** [comment]
I go on getting this type of message (not always):

Last follow is not verified.      ~'xxxxxxx' might be temporarily blocked from following

But not always, sometimes I get a correct follow process.

**Comment by ortegafernando** [comment]
See this:  [https://github.com/timgrossmann/InstaPy/issues/4893](https://github.com/timgrossmann/InstaPy/issues/4893)

**Comment by ortegafernando** [comment]
Anyone with this problem?

**Comment by ortegafernando** [comment]
https://github.com/timgrossmann/InstaPy/issues/5336

**Comment by sebbyG2** [comment]
This occurs because of the box which appears after the follow with the **Suggested** users. The script is checking for no 'Follow', but there are because of these **Suggested** users.

I'm not sure how to fix this in the code though. Is someone looking into this?

**Comment by RoberVF** [comment]
Hello guys. I had the same issue, but I think that the solution is too easy. You just need to open the web browser in all the screen to reduce the possibility to produce that error. Apart of that, the error solve by itself, I mean you just need to wait some minutes(4 or 5 depends) and it continue working!

**Comment by spacemishka** [comment]
have the same problem on windows....

**Comment by Dacaramo** [comment]
same problem guys I have tried what @RoberVF  said but it doesnt work I keep geting the error

**Comment by dipiana** [comment]
> This occurs because of the box which appears after the follow with the **Suggested** users. The script is checking for no 'Follow', but there are because of these **Suggested** users.
> 
> I'm not sure how to fix this in the code though. Is someone looking into this?

This sounds reasonable. Any idea how to fix it?

**Comment by pxp920** [comment]
The solution that @RoberVF offers is a solution for a separate issue.  I believe instagram is identifying that the instagram instance is running by a bot and restricting the activity, meaning it stops the activity and pops up the restriction message, however when you log in through your iphone normally you see that you are not, infact, banned. 

Any solution?

**Comment by BuZZy1337** [comment]
@pxp920 @dipiana I found a solution...
Change the instapy/xpath_compile.py file on Line 80-87.
https://github.com/timgrossmann/InstaPy/blob/cac8f713937647b89a02894d1e4121d38de2085c/instapy/xpath_compile.py#L80-L87
to the following..
```
xpath["get_following_status"] = {
    "follow_button_XP": "//button[(text()='Following' or \
                                  text()='Requested' or \
                                  text()='Follow' or \
                                  text()='Follow Back' or \
                                  text()='Unblock') and not(ancestor::*/@role = 'presentation')]",
    "follow_span_XP_following": "//button/div/span[contains(@aria-label, 'Following')]",
}
```
Save the file and enjoy...
👍

**Comment by neerajHebbar** [comment]
> @pxp920 @dipiana I found a solution...
> Change the instapy/xpath_compile.py file on Line 80-87.
> https://github.com/timgrossmann/InstaPy/blob/cac8f713937647b89a02894d1e4121d38de2085c/instapy/xpath_compile.py#L80-L87
> 
> 
> to the following..
> ```
> xpath["get_following_status"] = {
>     "follow_button_XP": "//button[(text()='Following' or \
>                                   text()='Requested' or \
>                                   text()='Follow' or \
>                                   text()='Follow Back' or \
>                                   text()='Unblock') and not(ancestor::*/@role = 'presentation')]",
>     "follow_span_XP_following": "//button/div/span[contains(@aria-label, 'Following')]",
> }
> ```
> 
> Save the file and enjoy...
> 👍

Its not working for me please help....

**Comment by neerajHebbar** [comment]
Still it is not liking or commenting.... 

> > @pxp920 @dipiana I found a solution...
> > Change the instapy/xpath_compile.py file on Line 80-87.
> > https://github.com/timgrossmann/InstaPy/blob/cac8f713937647b89a02894d1e4121d38de2085c/instapy/xpath_compile.py#L80-L87
> > 
> > to the following..
> > ```
> > xpath["get_following_status"] = {
> >     "follow_button_XP": "//button[(text()='Following' or \
> >                                   text()='Requested' or \
> >                                   text()='Follow' or \
> >                                   text()='Follow Back' or \
> >                                   text()='Unblock') and not(ancestor::

> @pxp920 @dipiana I found a solution...
> Change the instapy/xpath_compile.py file on Line 80-87.
> https://github.com/timgrossmann/InstaPy/blob/cac8f713937647b89a02894d1e4121d38de2085c/instapy/xpath_compile.py#L80-L87
> 
> 
> to the following..
> ```
> xpath["get_following_status"] = {
>     "follow_button_XP": "//button[(text()='Following' or \
>                                   text()='Requested' or \
>                                   text()='Follow' or \
>                                   text()='Follow Back' or \
>                                   text()='Unblock') and not(ancestor::*/@role = 'presentation')]",
>     "follow_span_XP_following": "//button/div/span[contains(@aria-label, 'Following')]",
> }
> ```
> 
> Save the file and enjoy...
> 👍

Please help it is not working I am able to like by changing the xpath but not able to comment or follow
